services/web-server/src/servers/createApp.js

```diff
@@ -31,7 +31,7 @@ export default async ({ cfg }) => {
 
   await Promise.all(
     cfg.app.loginStrategies.map(async strategy => {
-      const { default: loginStrategy } = await import(`../login/${strategy}`);
+      const { default: loginStrategy } = require(`../login/${strategy}`);
```

Comment:
`require` is a synchronous function. We would need to remove the `async` keyword from the function definition since we're not using `await` anymore. We would also need to remove `await Promise.all`.

I think you can just rewrite as:

```js
cfg.app.loginStrategies.forEach(strategy => {
  const { default: loginStrategy } = require(`../login/${strategy}`);

  loginStrategy(app, cfg);
});
```